tests/FINDOLOGIC/Export/Tests/DataHelperTest.php

```diff
@@ -100,4 +101,16 @@ public function numericValueProvider()
             'zero as string' => ['0', false]
         ];
     }
+
+    /**
+     * Test if character limit of data helper causes exception when called outside attribute class.
+     *
+     * @expectedException \FINDOLOGIC\Export\Helpers\AttributeValueLengthException
+     */
+    public function testCharacterLimitCausesException()
+    {
+        $value = implode('', array_fill(0, 16384, 'a'));
```

Comment:
See the above comment. Use a multibyte character for this test.

Reply:
Done :heavy_check_mark: